src/components/Dashboard/FilingHistoryList/filings/IncorporationApplication.vue

```diff
@@ -110,6 +111,13 @@ export default class IncorporationApplication extends Vue {
     )
   }
 
+  /** The legal name or numbered description of the new company. */
+  get companyName (): string {
+    if (this.getLegalName) return this.getLegalName
+    if (this.getEntityName) return `A ${this.getEntityName}`
+    return 'Unknown Name'
+  }
```

Comment:
This is a fix! Ie,

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/8cdfebec-f712-4501-999d-6d7714815e1c)